### DISCUSSION TITLE
Option to disable Auto AFK

### DISCUSSION THREAD

**Discussion by viasnake** [Zrips/CMI#4932] [Feature Requests]
I suggest that AutoAFK be completely disabled when AutoAfkIn is set to 0 or -1.

Currently, CMI can disable AFK but need to set permissions to disable only Auto AFK.
But, when controlled by permissions, Auto AFK will continue to apply only to administrators with wildcard permissions.
It is inconvenient to give False permissions to administrators just for Auto AFK.

**Comment by Zrips** [comment]
Will do with next update

**Reply by viasnake** [reply]
Thank you.
